tests/driver_dsp0401/main.c

```diff
@@ -49,19 +50,19 @@ int main(void)
     while (loop < LOOPS) {
         puts("[INFO] Displaying 'THIS IS RIOT'");
         dsp0401_display_text(&dev, (char*)"THIS");
-        xtimer_msleep(WORD_DELAY);
+        ztimer_sleep(ZTIMER_MSEC, WORD_DELAY);
```

Comment:
Since the driver requires `ZTIMER_USEC` anyway, why not use `usec` everywhere and avoid pulling in an unneeded timer?

Reply:
Done !